tools/ui-components/src/base.css

```diff
@@ -7,6 +7,21 @@
     @apply underline;
   }
 }
+@layer components {
+  /* Table component */
```

Comment:
I couldn't find a better way to do this (I tried applying the classNames in a React-y way with `React.Children` map but was unsuccessful.

Ref: 
![image](https://user-images.githubusercontent.com/27147016/184219972-4069ebbf-d6cd-482a-bd62-5bc1f10648c2.png)

Reply:
You should be able to that using tailwind variants:
https://tailwindcss.com/blog/tailwindcss-v3-1#arbitrary-values-but-for-variants

Reply:
Also looking into other design systems that are built with tailwind might help finding the best practices for styling the table and its child elements.